### PR TITLE
perf: Add global gflopss to operator log

### PR DESCRIPTION
**Pull request by georgebisbas**
Adds global gflops/s to the operator log

### CODE REVIEW

File: devito/operator/operator.py [hunk 1]
```diff
@@ -751,8 +751,12 @@ def _emit_apply_profiling(self, args):
 
             v = summary.globals.get('fdlike')
             if v is not None:
-                perf("%s* Achieved %.2f FD-GPts/s" % (indent, v.gpointss))
-
+                if v.gflopss is not None:
```
Comment (FabioLuporini) [inline devito/operator/operator.py:754]: remind me when it could be None ?

Comment (georgebisbas) [reply devito/operator/operator.py:754]: It happened when test_autotuning was running.
I will check again.

Comment (FabioLuporini) [reply devito/operator/operator.py:727]: OK, I see.

Can we show the GFlops/s first and the FD-GPts/s later? to make it homogeneous with the other prints

Comment (georgebisbas) [reply devito/operator/operator.py:727]: ok

**Review by FabioLuporini — COMMENTED** [review]
Please rebase and squash into a single commit

**Pull request merged**